firmware/console/binary/tunerstudio_outputs.h

```diff
@@ -229,12 +229,17 @@ typedef struct {
 	scaled_voltage rawOilPressure;		// 242
 
 	int16_t tuneCrc16; // 244
-    uint8_t sd_status; // 246
-	uint8_t pad;
 
-	scaled_voltage rawPpsSecondary;		// 248
+	uint8_t sd_status; // 246
```

Comment:
why'd these get moved around? these offsets are now also wrong, and need updating in `rusefi.input`

Reply:
easiest is just to put one value in place of the old `pad` (offset 247), then everything else in the unused chunk

Reply:
oh dear, didn't realize I did that